bio_phil/webapp/urls.py

```diff
@@ -1,14 +1,22 @@
 from django.contrib import admin
 from django.urls import path
 from . import views
+from django.contrib.staticfiles.urls import static
+from django.contrib.staticfiles.urls import staticfiles_urlpatterns
+from bio_phil import settings
 from django.contrib.auth.decorators import login_required
 
 urlpatterns = [
     path('', views.index, name='index'),
     path('register/', views.register, name='register'),
     path('generate/', views.generate_access_codes, name='generate_access_codes'),
     path('manage/', views.manage_access_codes, name='manage_access_codes'),
+    path('img_test',views.images,name ='image_tester'),
+    staticfiles_urlpatterns(),
+    static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```

Comment:
I don't think lines 15 & 16 need to be included. 

I run into this error when I try to make the migrations and run the server:
![screen shot 2018-09-16 at 10 21 47 pm](https://user-images.githubusercontent.com/15126359/45597483-af152e00-b9ff-11e8-9ef0-a416022466f4.png)

Commenting those lines out solved the issue for me